TLM/TLM/UI/MainMenu/DespawnButton.cs

```diff
@@ -11,8 +11,8 @@ public class DespawnButton : MenuButton {
 
         public override string Tooltip =>
             Options.disableDespawning
-                ? Translation.Options.Get("Maintenance.Button:Enable despawning")
-                : Translation.Options.Get("Maintenance.Checkbox:Disable despawning");
+                ? Translation.Menu.Get("Tooltip:Enable despawning")
+                : Translation.Menu.Get("Tooltip:Disable despawning");
```

Comment:
There is no string with key `Tooltip:Enable despawning` same for `Tooltip:Disable despawning`
It has to be introduced in Crowdin before we can accept this